### PR TITLE
Added final report and updated link in README.

### PR DESCRIPTION
**Pull request by ClintonCao**
I've added the final report, please check it before merging :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ This tool is for creating a visualization of the warnings/defects that were gene
 |Architecture Design  | [architecture design] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Product%20related%20documents/ArchitectureDesign(BlueTurtle).pdf)|
 |Product Requirement | [product requirement] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Requirements/RequirementsSoftwareVisualizationToolBlueTurtle.pdf)|
 |First Concept  | [first concept] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/First%20Concept/FirstConceptBlueTurtle.pdf) |
-|Final Report (Draft) | [final report] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Final%20Report/FinalReport(BlueTurtle).pdf) |
+|Final Report | [final report] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Final%20Report/FinalReport(BlueTurtle).pdf) |
```
Comment (SunweiWang) [inline README.md]: This link still gives me the draft.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,8 @@ This tool is for creating a visualization of the warnings/defects that were gene
 |Architecture Design  | [architecture design] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Product%20related%20documents/ArchitectureDesign(BlueTurtle).pdf)|
 |Product Requirement | [product requirement] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Requirements/RequirementsSoftwareVisualizationToolBlueTurtle.pdf)|
 |First Concept  | [first concept] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/First%20Concept/FirstConceptBlueTurtle.pdf) |
-|Final Report (Draft) | [final report] (https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Final%20Report/FinalReport(BlueTurtle).pdf) |
```
Comment (SunweiWang) [inline README.md]: And can you also change the text with url link also as: final report (draft)?

**Comment by ClintonCao** [comment]
Indeed it does, but the naming of the file (draft) was FinalReport(BlueTurtle). If you look at the new file names, it should be clear what I was trying to do with the links.

I will update the text for the draft.

**Comment by boninggong** [comment]
Final report looking good! We all already looked at it before uploading it here, so everything should be fine.

**Comment by ClintonCao** [comment]
Alright, I will merge this then.

**Pull request merged**